tests/medium/test_games.py

```diff
@@ -73,16 +73,8 @@ class SuperpowersInContainer(ContainerTests, test_games.SuperpowersTests):
     TIMEOUT_STOP = 10
 
     def setUp(self):
-        self.hosts = {443: ["github.com"]}
+        self.hosts = {443: ["api.github.com", "github.com"]}
         self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'superpowers')
         super().setUp()
         # override with container path
         self.installed_path = os.path.join(self.install_base_path, "games", "superpowers")
-
-    def test_install_with_changed_download_page(self):
```

Comment:
As you are overriding `get_metadata_and_check_license`, I still think it's important to override the download page in games and ides to fake the API returned values (seeding wrong content basically).